src/components/pages/public/sign-up/index.js

```diff
@@ -82,13 +86,14 @@ class SignUp extends Component {
                                 onClick={handleSubmit((user) => { createAccount(user); })}
                                 isLoading={isLoading}
                             />
-                            <p className="rainbow-auth-firebase-signup_terms-conditions">
-                                By creating an account you agree to our Terms and
-                                Conditions and our Privacy Policy.
-                            </p>
                         </article>
                     </Card>
-                    <Link className="rainbow-auth-firebase-signup_link" to="/home/signin">Sign in?</Link>
+                    <p className="rainbow-auth-firebase-signup_terms-conditions">
+                        By creating an account you agree to our<br/>
+                        <Link className="rainbow-auth-firebase-signup_link" to="/home/terms"> Terms and Conditions </Link>
+                        and our
+                        <Link className="rainbow-auth-firebase-signup_link" to="/home/policy"> Privacy Policy</Link>
+                    </p>
```

Comment:
here you need to fix the translation because now you need to split this sentence to translate it